src/main/java/org/jitsi/videobridge/octo/OctoRelay.java

```diff
@@ -41,16 +44,20 @@
 
     /**
      * The Octo relay ID which will be advertised by jitsi-videobridge.
+     * Other bridges can use this ID in order to discover the socket address
+     * that this bridge is accessible on. With the current implementation the
+     * ID just encodes a pre-configured IP address and port,
+     * e.g. "10.0.0.1:20000"
```

Comment:
They need to use the ID to find how to reach the bridge, but they might or might not use an extra mapping. For the first implementation I'm planning to use an address:port pair, but I wanted to leave it more abstract in case we need to change it later (e.g. to use a domain name, or something else)